x-pack/plugins/security_solution/public/flyout/entity_details/components/user_details_body.tsx

```diff
@@ -0,0 +1,155 @@
+  observedUser: ObservedUserData;
+  managedUser: ManagedUserData;
+  riskScoreState: RiskScoreState<RiskScoreEntity.user>;
+  contextID: string;
+  scopeId: string;
+  isDraggable: boolean;
+}
+
+export const UserDetailsBody = ({
+  userName,
+  observedUser,
+  managedUser,
+  riskScoreState,
+  contextID,
+  scopeId,
+  isDraggable,
+}: UserDetailsBodyComponentProps) => {
+  const { euiTheme } = useEuiTheme();
+
+  const lastSeenDate = useMemo(
+    () =>
+      max([observedUser.lastSeen, managedUser.lastSeen].map((el) => el.date && new Date(el.date))),
+    [managedUser.lastSeen, observedUser.lastSeen]
+  );
+
+  return (
+    <>
+      <EuiPanel hasShadow={false}>
+        <EuiFlexGroup gutterSize="s" responsive={false} direction="column">
+          <EuiFlexItem grow={false}>
+            <EuiText size="xs" data-test-subj={'user-details-body-lastSeen'}>
+              {lastSeenDate && <PreferenceFormattedDate value={lastSeenDate} />}
+              <EuiSpacer size="xs" />
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
+              <EuiFlexItem grow={false}>
+                <EuiIcon type="user" size="l" color="primary" />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiTitle size="s">
+                  <h2>
+                    <SecuritySolutionLinkAnchor
+                      deepLinkId={SecurityPageName.users}
+                      path={getUsersDetailsUrl(userName)}
+                      target={'_blank'}
+                      css={css`
+                        font-weight: ${euiTheme.font.weight.bold};
+                      `}
+                    >
+                      {userName}
+                    </SecuritySolutionLinkAnchor>
+                  </h2>
+                </EuiTitle>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup
+              gutterSize="s"
+              alignItems="center"
+              responsive={false}
+              data-test-subj="user-details-body-header"
+            >
+              <EuiFlexItem grow={false}>
+                {observedUser.lastSeen.date && (
+                  <EuiBadge data-test-subj="user-details-body-observed-badge" color="hollow">
+                    {i18n.OBSERVED_BADGE}
+                  </EuiBadge>
+                )}
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                {managedUser.lastSeen.date && (
+                  <EuiBadge data-test-subj="user-details-body-managed-badge" color="hollow">
+                    {i18n.MANAGED_BADGE}
+                  </EuiBadge>
+                )}
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiPanel>
+      <EuiHorizontalRule margin="none" />
```

Comment:
and if you update `margin="none"` to `margin="m"` you'll have the same UI, while having the code consistent with the `document_details` code

Reply:
When I do that, the horizontal bars no longer takes 100% of the width.
<img width="519" alt="Screenshot 2023-11-15 at 09 29 12" src="https://github.com/elastic/kibana/assets/1490444/1032c12c-cfa5-4106-9938-a03b5d98e61c">